Lib/uuid.py

```diff
@@ -626,13 +673,14 @@ def getnode():
         getters = [_unix_getnode, _ifconfig_getnode, _ip_getnode,
                    _arp_getnode, _lanscan_getnode, _netstat_getnode]
 
-    for getter in getters + [_random_getnode]:
+    for getter in getters:
```

Comment:
I think the only possibility is an import error.  Ultimately, `random.getrandbits()` boils down to `os.urandom()` which I think can block but AFAICT won't raise an exception.  And it bothers me (as I think it bothers you) that all exceptions in all those getters are just swallowed.  Here's my thinking: since I'm not sure of the best way to report any errors in the other getters, and since the last getter is `_random_getnode()`, and since it's pretty unlikely that the function will experience an `ImportError`, and if it *does* then I think it's better to report that than to silently swallow the exception and return `None` from `getnode()`, since that will lead to even more mysterious bugs, I think letting any rare exceptions in `_random_getnode()` *not* be swallowed is the better choice.

Reply:
It's LGTM. I just wanted to make sure that this is an intentional change.